config/database.yml

```diff
@@ -0,0 +1,20 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
+  # https://guides.rubyonrails.org/configuring.html#database-pooling
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+
+development:
+  <<: *default
+  database: flashsale_development
+  username: postgres
+  password: 1234
+  host: localhost
+  port: 5432
+
+production:
+  <<: *default
+  database: flashsale_production
+  username: postgres
+  password: 1234
```

Comment:
let's not expose username and password in the code. Find a way to hide these

Reply:
done